packages/bpk-tokens/src/web/base/ratings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "imports": [
+    "./aliases.json"
+  ],
+  "global": {
+    "category": "ratings"
+  },
+  "props": {
+    "RATINGS_BASE_BORDER_RADIUS": {
+      "value": "2.625rem",
```

Comment:
Where did this value come from? We typically have an extra layer of indirection here. For example:

In `buttons.json` we use `BORDER_RADIUS_PILL_LG` when defining `BUTTON_BORDER_RADIUS_LG`: https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/src/web/base/buttons.json#L26)

Then, the source of truth for `BORDER_RADIUS_PILL_LG` is here: https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/src/web/base/aliases.json#L24

So in this case we should be able to do a similar thing where you define a value in `aliases.json` and then use it in this file as a variable.

As an aside, is it not possible to use one of the existing border radius tokens in here instead of defining your own? I imagine `BORDER_RADIUS_PILL_LG` might work for your use case.

Reply:
Resolved through use of `border-radius: 50%`